react-leaflet/build.boot

```diff
@@ -1,14 +1,14 @@
 (set-env!
   :resource-paths #{"resources"}
   :dependencies '[[cljsjs/boot-cljsjs "0.7.0" :scope "test"]
-                  [cljsjs/react "15.6.1-1"]
-                  [cljsjs/react-dom "15.6.1-1"]
+                  [cljsjs/react "15.6.1-2"]
+                  [cljsjs/react-dom "15.6.1-2"]
                   [cljsjs/leaflet "1.1.0-2"]])
 
 (require '[cljsjs.boot-cljsjs.packaging :refer :all])
 
-(def +lib-version+ "1.4.1")
-(def +version+ (str +lib-version+ "-2"))
+(def +lib-version+ "1.6.5")
+(def +version+ +lib-version+)
```

Comment:
+version+ must include the build identifier.

Reply:
OK! I added that now. I'm assuming it is for the current `+lib-version+`, so `-1` in this case?